daml-foundations/daml-ghc/daml-stdlib-src/DA/Internal/LF.daml

```diff
@@ -180,3 +183,11 @@ submitMustFail = primitive @"SMustFailAt"
 -- | Declare you are building a scenario.
 scenario : Scenario a -> Scenario a
 scenario = identity
+
+-- | HIDE A dummy type for the DAML-LF structural record type
+-- `<f1: a1, f2: a2>`.
+data Pair (f1 : Symbol) (f2 : Symbol) a1 a2 = Pair Opaque
```

Comment:
I am slightly confused here. We don’t seem to convert to this type at any point. Is that left for a separate PR? If so, it might be nice to add a comment here in the meantime.